app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -37,9 +37,15 @@ Teach in England and you’ll benefit from working in a world-class education sy
 
 You’ll earn a [competitive salary](/life-as-a-teacher/pay-and-benefits/teacher-pay), get supported in your professional development and have varied job opportunities.
 
-You could also consider teaching in [Wales](https://www.ewc.wales/site/index.php/en/registration/index.php?option=com_content&view=article&id=12&Itemid=170&lang=en), [Scotland](https://teachinscotland.scot/) or [Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
+You could also consider teaching in other areas of the UK, including:
```

Comment:
I found this quite difficult to read - could you consider something like the below? 
```suggestion
To find out about teaching in other areas of the UK, visit the:
```
Then could change the bullets to:

* Educator Wales website
* Teach in Scotland website
* Northern Ireland Department for Education website